Avo.ts

```diff
@@ -1642,21 +1696,23 @@ export interface DocsPageViewedProperties {
   utmMedium: string | null | undefined;
   utmContent: string | null | undefined;
   utmCampaign: string | null | undefined;
+  userId_: string;
   referrer: string | null | undefined;
   path: string;
 }
 /**
- * Docs Page Viewed: Sent when user opens one of the docs pages.
+ * Docs Page Viewed: Sent when user opens one of the docs pages (which is not contained in our Landing Page Viewed event).
  *
  * @param properties the properties associatied with this event
  * @param properties.utmSource: Parsed from the link a user clicks to get to Avo.
  * @param properties.utmMedium: Parsed from the link a user clicks to get to Avo
  * @param properties.utmContent: Parsed from the link a user clicks to get to Avo
- * @param properties.utmCampaign: Parsed from the link a user clicks to get to Avo.
+ * @param properties.utmCampaign: Identifies a specific product promotion or strategic campaign this event is part of
+ * @param properties.userId_: The value used to identify the user. Make sure it's a unique sequence of characters used to identify the user.
```

Comment:
Codegen wants this new property _userId_ but—and correct me if I'm wrong—it's not used a all in the code! Do you know what might be happening @alex-tpom6oh?

Reply:
Actually I can't see the referrer being used either... These both belong to the UpdateUserProperties action: https://www.avo.app/schemas/fwtXqAc0fCLy7b7oGW40/branches/0vRisgo5p/e/J4cW6PmriY

Reply:
> referrer

It's because of setOnce, only Amplitude has the setOnce support so user props are only sent there

> userId_

It's an unused enrichment, looks like a bug, you can ignore it for now.